docs/03-03-examples-alpha.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Kyma CLI alpha command usage examples
+type: Details
+---
+
+The following examples show how to use the alpha commands to provision a cluster, install and upgrade Kyma, and handle errors.
+
+## Provision a cluster
+To provision a Kubernetes cluster, run:
+
+```
+kyma alpha provision k3s 
+```
+If you want to define the name of your cluster and pass arguments to the Kubernetes API server (for example, to log to standard errors), run:
+
+```
+kyma alpha provision k3s --name='custom_name' --server-args='--alsologtostderr'
+```
+
+
+## Install Kyma
+
+There are several ways to install Kyma:
+
+- To install Kyma on a local cluster, simply use the `deploy` command without any flags.
+Kyma provides a default domain under the URL `https://console.local.kyma.dev`.
+
+  ```
+  kyma alpha deploy 
+  ```
+
+- You can also install Kyma using your own domain name, or on a cluster that does not run locally. <br>
```

Comment:
So just to be clear. The domain is not a requirement for a non-local cluster, there are just different default behaviours

Reply:
```suggestion
- If you prefer to install Kyma using your own domain name, you must provide the certificate and key as files. 
```